### PR TITLE
Silence logger during E2E tests

### PR DESCRIPTION
**Pull request by aunyks**
Fixing #522.

### CODE REVIEW

**Comment by aunyks** [comment]
Not very sure what Labeler is communicating.

**Review by mpetrunic — COMMENTED** [review]
this works, although better solution might be to just go in e2e tests and silence logger there or create mocked logger. Reasoning is that we use IDE shortcuts to run tests (or subset) and this would require us to configure env variable in IDe or console.

File: packages/lodestar/test/e2e/network/network.test.ts [hunk 1]
```diff
@@ -27,11 +27,12 @@ describe("[network] network", function () {
   this.timeout(5000)
   let netA: Libp2pNetwork, netB: Libp2pNetwork;
   const logger: ILogger = new WinstonLogger();
-  const metrics = new BeaconMetrics({enabled: true, timeout: 5000, pushGateway: false}, {logger});
+  logger.silent = true;
```
Comment (aunyks) [inline packages/lodestar/test/e2e/network/network.test.ts:30]: @mpetrunic Line 30 should silence the logger during E2E without relying just on the envar.

**Review by mpetrunic — DISMISSED** [review]

File: packages/lodestar/src/logger/winston.ts [hunk 2]
```diff
@@ -47,7 +47,7 @@ export class WinstonLogger implements ILogger {
     });
     //@ts-ignore
     this._level = LogLevel[options.level];
-    this._silent = false;
+    this._silent = process.env.LODESTAR_SILENCE === 'true';
```
Comment (wemeetagain) [inline packages/lodestar/src/logger/winston.ts]: Can you add a documentation note somewhere, maybe the README or elsewhere, about the `LODESTAR_SILENCE` env var?

Another thought, can you do a check for if `process` exists? I think in the browser `process` will be `undefined`

**Review by wemeetagain — CHANGES_REQUESTED** [review]

File: packages/lodestar/src/logger/winston.ts (hunk 2 above)
Comment (aunyks) [reply packages/lodestar/src/logger/winston.ts]: Good eye on `process` possibly being undefined. I'll update this logic and make note of the envar in the README.

**Review by wemeetagain — DISMISSED** [review]
LGTM :)

**Review by mpetrunic — DISMISSED** [review]

**Comment by mpetrunic** [comment]
@aunyks you have lint errors, check CI

**Review by mpetrunic — APPROVED** [review]

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**